src/components/MyBoard/index.js

```diff
@@ -1,24 +1,13 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import Container from 'react-bootstrap/Container';
-import Table from 'react-bootstrap/Table';
-
-import Body from './Body';
-import Header from './Header';
-
-function MyBoard({ N }) {
-    return (
-        <Container>
-            <Table striped bordered hover size="sm">
-                <Header N={N} />
-                <Body N={N} />
-            </Table>
-        </Container>
+import TurnTable from '../TurnTable';
+import data from '../../data/2_board.json';
+import './styles.css';
+
+function MyBoard() {
+    return (<div>
+        <TurnTable header={'My'} turnRecords={data.board.myturns}/>
```

Comment:
try replacing `header={'My'}` with `isMine`
`<TurnTable isMine turnRecords={...} />`

then you're in TurnTable
`const header = (props.isMine) ? 'My Board' : 'Your Board'`